ccpp/suites/suite_FV3_GFS_v15_thompson_mynn_noahmp.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<suite name="FV3_GFS_v15_thompson_mynn_noahmp" lib="ccppphys" ver="4">
```

Comment:
Is this file required? It's not related to flake. If it is, please fix line 3 in this SDF and all others in this PR. The syntax `lib="ccppphys" ver="4"` is obsolete (since several months already!) and needs to be replaced with just `version=1`.